src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = (props) => {
+  const { name, age, sex, isMarried, partnerName } = props.person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+
+      <p className="Person__partner">
+        {isMarried && (`${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`)}
+        {!isMarried && ('I am not married')}
```

Comment:
A perfect use-case for a ternary operator here:
```suggestion
        {
          isMarried
            ? (`${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`)
            : ('I am not married')
        }
```

Not an impactful change for performance, but this way, we are evaluating `isMarried` only once, instead of twice